airesources/Julia/hlt/entity.jl

```diff
@@ -103,66 +103,38 @@ struct Ship <: Entity
     docked_planet_id::String
     progress::Int
     cooldown::Int
-    
-    function Ship(owner_id::String, tokens::Vector{String})
-        id = shift!(tokens)
-        x = parse(Float64, shift!(tokens))
-        y = parse(Float64, shift!(tokens))
-        hp = parse(Int, shift!(tokens))
-        vel_x = parse(Float64, shift!(tokens))
-        vel_y = parse(Float64, shift!(tokens))
-        docked = DockedStatus(parse(Int, shift!(tokens)))
-        docked_planet = shift!(tokens)
-        progress = parse(Int, shift!(tokens))
-        cooldown = parse(Int, shift!(tokens))
-
-        new(owner_id, id, x, y, hp, Constants.SHIP_RADIUS, vel_x, vel_y, docked, docked_planet, progress, cooldown)
-    end
-
 end
-
-"""
-    isdocked(ship)
-
-Test if current ship is undocked. Returns `true` if ship is `UNDOCKED`, `false` otherwise.
-"""
-isdocked(ship::Ship) = ship.docked != UNDOCKED
+function Ship(owner_id::String, tokens::Vector{String})
+    id = shift!(tokens)
+    x = parse(Float64, shift!(tokens))
+    y = parse(Float64, shift!(tokens))
+    hp = parse(Int, shift!(tokens))
+    vel_x = parse(Float64, shift!(tokens))
```

Comment:
Can you actually drop the vel_x and vel_y? They are deprecated so having those variables in the kit might be confusing to people.

Reply:
(I.e. still read them since they are still part of the protocol with game engine, but don't store the values in any variables.)

Reply:
yeah of course i will. I just edited the things that seemed obviously strange to me.

Reply:
Yup, just taking it as opportunity to clean it up a bit more.